<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(230,230,230);

}

QPushButton {
	background-color: rgb(255,255,255);  /* Background color */
    color: rgb(0, 0, 150);  /* Text color */
    border: 2px solid rgb(255,155, 50);  /* Border color */
    border-radius: 5px;  /* Rounded edges */
	padding: 2px;
}

QPushButton:hover {
    background-color: rgb(245, 245, 255);  /* Background color when hovered */
	color: rgb(0, 0, 200);
    border-color:rgb(225, 105, 0);  /* Border color when hovered */
}

QPushButton:pressed {
   background-color: rgb(200,200, 210);  /* Background color when pressed */
	border: 3px solid rgb(205, 85, 0);
}
QPushButton:clicked {
   background-color: rgb(200, 200, 210);  /* Background color when pressed */
	border:  rgb(205, 85, 0);
}

QPushButton:checked {
    background-color: rgb(220,120, 15);
	border: 3px solid rgb(205, 105, 0);
	font-weight: bold;  /* Bold text */
}

QPushButton:disabled {
    background-color: rgb(220,220, 220);
	border: 3px solid rgb(205, 105, 0);
	font-weight: bold;  /* Bold text */
}

QDoubleSpinBox:enabled{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}
QDoubleSpinBox:disabled{
color: rgb(0, 0, 200);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}

QSpinBox:enabled{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}
QSpinBox:disabled{
color: rgb(0, 0, 200);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}

QGroupBox{
color: rgb(0, 0, 150);  /* Text color */
font-weight: bold;  /* Bold text */
}

QLabel{
color: rgb(0, 0, 150);  /* Text color */
}

QPlainTextEdit{
background-color: rgb(255,255,255);
border-radius: 5px;  /* Rounded edges */
}

QLineEdit{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 3px solid rgb(255,155,50);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QComboBox{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 3px solid rgb(255,155,50);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QTabBar::tab {
    color: rgb(0, 0, 150); 
    border: 2px solid rgb(255,155,50);
    padding: 5px;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
	min-width:80px;
}

QTabBar::tab:selected {
    color:rgb(0, 0, 150); 
    background-color: rgb(255, 255, 255); ;
    font-weight: bold;
	border: 3px solid rgb(235, 135, 30);
}

QTabBar::tab:hover {
    color: rgb(0, 0, 200); /* Slightly lighter orange */
	background-color: rgb(245, 245, 255); ;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,0,1">
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="QLineEdit" name="filename_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_file_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>20</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resources/load.png</normaloff>resources/load.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>NC-File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Process Step Name: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="step_name_edit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,1,0,1" columnstretch="1,1,1,0,0">
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>WP: Y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="wp_y_spinbox">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QDoubleSpinBox" name="wp_x_spinbox">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>WP: Z</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>WP: X</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="rel_y_spinbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="wp_z_spinbox">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Rel. X:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QDoubleSpinBox" name="rel_x_spinbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Rel: Z</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Rel: Y</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QDoubleSpinBox" name="rel_z_spinbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QPushButton" name="set_current_pos_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resources/crosshair.png</normaloff>resources/crosshair.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="QPushButton" name="remove_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>resources/trash.png</normaloff>resources/trash.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
